hs_core/views/__init__.py

```diff
@@ -339,6 +339,31 @@ def delete_resource(request, shortkey, *args, **kwargs):
 
     return HttpResponseRedirect('/my-resources/')
 
+
+def rep_res_bag_to_irods_user_zone(request, shortkey, *args, **kwargs):
+    if request.is_ajax:
```

Comment:
Suggest to change the function name for readability. From the name of the function can't determine the purpose of the function. Also adding docstring would help.